core/src/data/dataSource.cpp

```diff
@@ -7,7 +7,7 @@
 #include "tile/tileManager.h"
 #include "labels/labels.h"
 
-//---- DataSource Implementation----
+namespace Tangram {
```

Comment:
Actually I've just realized that these are *not* totally equivalent. With 'using', anything not in a class must have its name prefixed by the relevant namespace.